controller/controller_interface.py

```diff
@@ -12,6 +12,33 @@ def get_controller_driver() -> ControllerDriver:
     return ControllerDriver(settings.ROOT_CONTROLLER_URL, token=token, aio_channel=True)
 
 
-def get_controller_status() -> Description:
-    """Get the controller status."""
-    return get_controller_driver().get_status()
+def get_controller_status() -> Status:
+    """Get the controller state.
+
+    Returns:
+        str: The controller state.
+    """
+    return get_controller_driver().get_status().data
+
+
+def get_fsm_state() -> str:
+    """Get the finite state machine state.
+
+    Returns:
+        str: The state the FSM is in.
+    """
+    return get_controller_status().state
+
+
+def send_event(event: str, **kwargs: dict[str, str]) -> FSMResponseFlag:
+    """Send an event to the controller.
+
+    Args:
+        event (str): The event to send.
+        **kwargs (dict[str, str]): The arguments for the event.
```

Comment:
No need to repeat the type info in the docstring. Mkdocs can get it from the type hint.